modules/__tests__/LocationUtils-test.js

```diff
@@ -160,3 +160,71 @@ describe("createLocation", () => {
     });
   });
 });
+
+describe("shouldReplace", () => {
+  it("should return true if current path is the same than new path", () => {
```

Comment:
Engilsh: `same as` or `other than`, `than` is exclusive, where `as` is inclusive. This is repeated in each test description.

Reply:
Ok, done. Thanks.